charts/sftd/README.md

```diff
@@ -0,0 +1,93 @@
+# SFTD Chart
+
+## Deploy
+
+
+Using your own certificates:
+
+```
+helm install sftd charts/std --set host=example.com --set-file tls.crt=/path/to/tls.crt --set-file tls.key=/path/to/tls.key
+```
+
+Using Cert-manager:
+```
+helm install sftd charts/std --set host=example.com --set tls.issuerRef.name=letsencrypt-staging
+```
+
+You can switch between `cert-manager` and own-provided certificates at any
+time. Helm will delete the `sftd` secret automatically and then cert-manager
+will create it instead.
+
+
+## DNS
+
+An SRV record needs to be present for the domain name defined in `host`.
+This is an artifact of the fact the wire backend currently uses public SRV
+records for service discovery. The record needs to be of the form:
+
+```
+_sft._tcp.{{ .Values.host }}  {{ .Values.host }} 443
+```
```

Comment:
question: how does brig learn about SFT FQDNs in a k8s scenario, since there is no srv-announcer? (AFAIK hard-coded as part of the configmap, at least that is how it is with restund) 

thought: In my understanding, this part is tight to how we run SFT atm, which is w/o K8s. SO, I'd say this information is obsolete in a k8s scenario, or am I missing sth here?

Reply:
Good question:

So ideally brig would just have an `sftStatic` field just like we have for TURN; that points to the loadbalancer. That code doesn't exist yet but there is a ticket in JIRA to write it.

For now; we ask operators to set up a single SRV record manually that simply points to the `A` record of the same name. That `A`  record will point to the ingress (which is loadbalanced). So there is no need to dynamically update this SRV record as it will always have just one entry.